.travis.yml

```diff
@@ -33,15 +33,18 @@ before_script:
   # Install catkin_tools test harness dependencies
   - ./.travis.before_script.bash
   - pip install empy sphinx_rtd_theme nose coverage flake8 mock --upgrade
+  - python2 -m pip install --user catkin-pkg
```

Comment:
Is this necessary? Pretty sure we should always just call `python` and let the PATH set by travis steer that to the right place.

Reply:
Building catkin with python 2 requires the catkin_pkg package for python 2. This is not related to the python version for catkin_tools which is managed by travis